plane/src/drone/docker/mod.rs

```diff
@@ -235,9 +244,9 @@ impl PlaneDocker {
             {
                 Ok(result) => {
                     let num_images_deleted = result.images_deleted.map(|d| d.len()).unwrap_or(0);
-                    tracing::info!(num_images_deleted, "Removed dangling images.");
+                    tracing::info!(num_images_deleted, "Pruning images.");
```

Comment:
"dangling" has a special meaning in docker and we don't necessarily abide by it here